From 0dc9a5275327952e871e8b507c9982392fb00d75 Mon Sep 17 00:00:00 2001
From: Anish Trivedi <anish@freescale.com>
Date: Thu, 31 Mar 2011 15:17:09 -0500
Subject: [PATCH 216/505] ENGR00141477 [imx-test] SAHARA: km_test_init should
 return 0 upon success, not 1

Upon success, return 0. Upon failure return a negative error code.

Signed-off-by: Anish Trivedi <anish@freescale.com>
---
 module_test/km_test.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/module_test/km_test.c b/module_test/km_test.c
index b317487..b879b53 100644
--- a/module_test/km_test.c
+++ b/module_test/km_test.c
@@ -1,5 +1,5 @@
 /*
- * Copyright 2005-2006 Freescale Semiconductor, Inc. All Rights Reserved.
+ * Copyright 2005-2006, 2011 Freescale Semiconductor, Inc. All Rights Reserved.
  */
 /*
  * The code contained herein is licensed under the GNU General Public
@@ -43,10 +43,10 @@ OS_DEV_INIT(km_test_init)
 
 	if (code == OS_ERROR_FAIL_S) {
 		os_printk("km_test driver initialization failed: %d\n", code);
-		retval = FALSE;
+		retval = code;
 	} else {
 		Major = code;
-		retval = TRUE;
+		retval = 0;
 	}
 
 	os_dev_init_return(retval);
-- 
1.8.0

